pkg/vm/engine/types.go

```diff
@@ -143,6 +145,8 @@ type Relation interface {
 	AddTableDef(context.Context, TableDef) error
 	DelTableDef(context.Context, TableDef) error
 
+	GetTableID(context.Context) uint64
```

Comment:
Why do you need this? I think all table definitions should go TableDefs() rather than adding random api's.

Reply:
done